html-templates.html

```diff
@@ -1,115 +1,157 @@
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>HTML Templates</title>
+        <link rel="stylesheet" href="./css/pages/home.css" />
+        <link rel="stylesheet" href="./css/components/header.css" />
+        <link rel="stylesheet" href="./css/components/buttons.css" />
+        <link rel="stylesheet" href="./css/components/slider.css" />
+        <link rel="stylesheet" href="./css/components/feature-card.css" />
+        <link rel="stylesheet" href="./css/components/progress-bar.css" />
+        <link rel="stylesheet" href="./css/components/feature-box.css" />
+        <link rel="stylesheet" href="./css/components/forms.css" />
+        <link rel="stylesheet" href="./css/components/blog-card.css" />
+        <link rel="stylesheet" href="./css/components/testimonial.css" />
+        <link rel="stylesheet" href="./css/components/socials.css" />
+        <link rel="stylesheet" href="./css/components/footer.css" />
+    </head>
+
+    <body>
+        <!-- HEADER-start -->
+        <!-- HEADER-end  -->
+
+        <!-- BUTTONS-start  -->
+        <!-- primary button -->
+        <a class="primary-btn" href="#">Get Started</a>
+        <!-- secondary button -->
+        <a class="secondary-btn" href="#">Digital Technology</a>
+        <!-- BUTTONS-end  -->
+
+        <!-- SLIDER-start  -->
+        <!-- light -->
+        <!-- dark -->
+        <!-- SLIDER-end  -->
+
+        <!-- FEATURE-CARD-start  -->
+        <div class="feature-card">
+            <img
+                src="./img/about-us-1/1.png"
+                class="feature-icon"
+                alt="Laptop illustration"
+            />
+            <h5 class="feature-title">High-quality Code</h5>
+            <p class="feature-des">
+                At vero eos et accusamus etiusto odio praesentium accusamus.
+            </p>
+        </div>
+        <!-- FEATURE-CARD-end  -->
+
+        <!-- PROGRESS-BAR-start  -->
+        <div class="progress-bar">
+            <div class="top">
+                <p class="label">Software Development</p>
+                <p class="value">92%</p>
+            </div>
+            <div class="bottom">
+                <div class="progress" style="width: 92%">
+                    <div class="bar"></div>
+                </div>
+            </div>
+        </div>
+        <!-- PROGRESS-BAR-end  -->
+
+        <!-- FEATURE-BOX-start -->
+        <div class="feature-box">
+            <div class="left-column">
+                <img src="./img/why-us/1.png" alt="First Growing Process" />
+            </div>
+            <div class="right-column">
+                <h5 class="title">First Growing Process</h5>
+                <p class="text">
+                    At vero eos et accusamus etiusto odio praesentium.
+                </p>
+            </div>
+        </div>
+        <!-- FEATURE-BOX-end  -->
+
+        <!-- FORMS-start  -->
```

Comment:
Somewhere in between should be blog card

Reply:
card is fine by the way

Reply:
I think maybe you forgot to do `git merge origin` after my blog-card pull request came through. There are conflicts now as seen at the bottom of this page.